src/ImageProcessorCore/Formats/Bmp/README.md

```diff
@@ -1,8 +1,8 @@
-﻿Encoder/Decoder adapted from:
-
-https://github.com/yufeih/Nine.Imaging/
-https://imagetools.codeplex.com/
-
-TODO:
-
-- Add support for all bitmap formats.
+﻿Encoder/Decoder adapted from:

```

Comment:
I use Git for Windows to manage my repo. I find that it's really easy to spot and discard changes. using it.